packages/tldraw/src/lib/utils/static-assets/assetUrls.ts

```diff
@@ -1,14 +1,5 @@
-import { RecursivePartial } from '@tldraw/editor'
+import { RecursivePartial, getDefaultCdnBaseUrl } from '@tldraw/editor'
 import { useMemo } from 'react'
-import { version } from '../../ui/version'
-
-/** @internal */
-const CDN_BASE_URL = 'https://cdn.tldraw.com'
-
-/** @public */
-export function getDefaultCdnBaseUrl() {
```

Comment:
Moved this to the editor package since it also needs it, didn't find a better spot for it.